org-ql.el

```diff
@@ -825,6 +825,47 @@ A and B are Org headline elements."
             (a-priority t)
             (b-priority nil)))))
 
+;;;; Functions/Occur
+
+(defun org-ql-occur (query &optional keep-previous callback)
```

Comment:
Since we don't need the callback (see below), we could add an optional `buffer` argument, defaulting to `(current-buffer)` with `cl-defun`.  That way users could call it on a certain buffer from a custom command.  We might need to add code to display the buffer if it's not already visible, but I think that would make the command even more useful.

It's not a bad idea to mirror the `org-occur` command, but I think that we can improve it and make it easier to use.  In fact, I'd recommend keyword arguments for `:buffer` and `:keep-previous`, in case we add more arguments in the future.

Reply:
Okay, please feel free to change the API.

Reply:
Fixed.